.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,8 +1,15 @@
+To ease the review process, please consider the following before to open the pull request:
+- is your code well commented?
+- did you update the relevant README(s)? See e.g. https://github.com/key4hep/k4geo/blob/main/detector/calorimeter/README.md or https://github.com/key4hep/k4geo/blob/main/FCCee/IDEA/compact/README.md
+- if you added some code, is is covered by tests? See https://github.com/key4hep/k4geo/blob/main/lcgeoTests/CMakeLists.txt
+- is your PR branch up to date with origin 'main'?
+- are you not adding/modifying files that should not have been added/modified (or at least, not in this PR)?
 
+If your modifications are compliant with the above: delete thoses lines from the PR description, populate the release note text below, and open the PR!
```

Comment:
I think you can make this a comment block using `<!--` and `-->`. That way the text would still show up for people if they open a PR, but it would not clutter the PR once it has been submitted, because it won't be visible. This obviously depends a bit on whether you want to make it harder to ignore or not ;)

Reply:
I am also not sure if PRs also have the fancy features that issues have (see, e.g. [DD4hep issues](https://github.com/AIDASoft/DD4hep/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml))

Reply:
Yes, I think we can make it a comment block (it is anyway easy to ignore :-P )

Reply:
People are going to ignore it anyway, so if we make it checkboxes, we can see from the PR overview that we can also ignore the PR!